7-React/src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,44 @@
+import React,{useState} from "react";
+import classes from "./TodoItem.module.css";
+import dataHooks from "./data-hooks";
+
+const TodoItem = ({
+  title,
+  deleteTaskHandler,
+    editTaskHandler,
+}: {
+  title: string;
+  deleteTaskHandler: () => {};
+        editTaskHandler: () => {};
+  
+    }) => {
+    
+    const [isDone, setIsDone] = useState(false);
+    const toggleHandler = () => {
+        setIsDone(prevState => !prevState);
+        
+    }
+  return (
+    <li data-hook={dataHooks.root} className={classes["to-do-tasks"]}>
```

Comment:
even tho the class name is `to-do-tasks` you can access it with `classes.toDoTasks`. this will ensure that if someone might change it in the future, you will get error